src/main/java/io/sudhakar/student/controller/GenericQueryController.java

```diff
@@ -21,7 +21,6 @@ public GenericQueryController(GenericService genericService) {
         this.genericService = genericService;
     }
 
-
     @GetMapping
     public ResponseEntity<StudentResponse<List<Student>>> getAllStudents(@RequestParam("queries") String queryName) {
```

Comment:
@RequestParam("queries") change to 
query

Reply:
chage name to query